server.js

```diff
@@ -1,37 +1,159 @@
+/* eslint-disable max-len */
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+// import dotenv from 'dotenv'
+import listEndpoints from 'express-list-endpoints'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import btc from './data/btc.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+// dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/btc"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const btcHistoricalSchema = new mongoose.Schema({
+  SNo: Number,
+  Name: String,
+  Symbol: String,
+  Date: String,
+  High: Number,
+  Low: Number,
+  Open: Number,
+  Close: Number,
+  Volume: Number,
+  Marketcap: Number
+});
+
+const BtcHistorical = mongoose.model('BtcHistorical', btcHistoricalSchema);
+
+if (process.env.RESET_DB) {
+  const seedDB = async () => {
+    await BtcHistorical.deleteMany();
+
+    await btc.forEach((item) => {
+      const newBtcHistorical = new BtcHistorical(item);
+      newBtcHistorical.save();
+    });
+  }
+  seedDB();
+}
+
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/', (_, res) => {
+  res.send(listEndpoints(app));
 })
 
+app.get('/btc-historical-prices', async (req, res) => {
```

Comment:
Maybe add a comment here on what this endpoint do just to be consistent with the comments=)